### ISSUE TITLE
Suggestion: Disambiguate providers

### ISSUE THREAD

**Issue by danb35** [fastenhealth/fasten-onprem#507]
Many providers/practices have similar, if not identical, names, and I don't see that the Fasten user interface provides a way to disambiguate them.  As just one example, I was searching for my gastroenterologist's practice, Gastroenterology Consultants PC.  I see two practices returned by that name, neither of which has a logo, and no information about city/state, website, portal URL, or any other way to distinguish them (and as it turns out, neither of them was the one I was looking for).  Searching by URL as suggested in #497 would certainly help with this (and may ultimately be the best solution), but so would providing this kind of other identifying information about the providers/practices listed.

**Comment by dwradcliffe** [comment]
+1 yes this is very confusing

**Comment by AnalogJ** [comment]
agreed. We have a tool that allows patients to customize and provide additional data about healthsystems - https://toolbox.fastenhealth.com/catalog/editor

However as you've noticed, we don't display all that metadata in the UI. This is partially because it would make the UI more complicated -- but it's mostly because the data is super sparse. a large number of health systems are missing websites and address information. I generated our health system catalog manually, and as you can tell it was pretty difficult to get accurate logos and websites for every entry.

Once HTI-1 (the updated version of the Cures Act Final Rule) comes into effect in Dec 2024, health systems will be required to publish this data themselves, so I expect the quality of entries in our catalog to improve, and we should be able to display those additional fields (and make them searchable).

I'll leave this open as an enhancement.

**Comment by AnalogJ** [comment]
This is now being actively worked on. 
It's not perfect, but so far we have location information for 90% of brands. 
We still only have websites for 50% of brands -- though that should get better as we start updating with HTI1 data.


Once we hit ~75% I'll update the search to allow website and state filtering.
